src/components/Personalization/groupDecisions.js

```diff
@@ -86,14 +87,17 @@ const isViewScope = scopeDetails =>
   scopeDetails.characteristics.scopeType &&
   scopeDetails.characteristics.scopeType === VIEW_SCOPE_TYPE;
 
```

Comment:
We could move this to `./utils`

Reply:
Yes, but Jon previously suggested we keep all surface-related utils in a single module